trace-encoder/src/main/java/com/squareup/tracing/TraceEncoder.kt

```diff
@@ -12,27 +12,24 @@ import kotlinx.coroutines.channels.consumeEach
 import okio.BufferedSink
 import java.io.Closeable
 import java.util.concurrent.atomic.AtomicInteger
-import kotlin.time.ExperimentalTime
-import kotlin.time.TimeMark
 import kotlin.time.TimeSource
 
 /**
  * Encodes and writes [trace events][TraceEvent] to an Okio [BufferedSink].
  *
  * @param scope The [CoroutineScope] that defines the lifetime for the encoder. When the scope is
  * cancelled or fails, the sink returned from [sinkProvider] will be closed.
- * @param start The [TimeMark] to consider the beginning timestamp of the trace. All trace events'
- * timestamps are relative to this mark.
+ * @param start The [TraceEncoderTimeMark] to consider the beginning timestamp of the trace. All
```

Comment:
It's still a `TimeMark`, no?

```suggestion
 * @param start The [TimeMark] to consider the beginning timestamp of the trace. All
```

Reply:
Tried to revert this change and I think I ended up dropping the commit. Will fix.